app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+  def welcome
+    @user = User.find(session[:user_id])
+  end
```

Comment:
Is it really needed?
You created this one 
https://github.com/Psiale/private-events/blob/3f4bd91f1c49ea28ce7da13f7b549c4ca00332bb/app/helpers/application_helper.rb#L1-L5